chia/rpc/wallet_rpc_client.py

```diff
@@ -105,10 +105,24 @@ async def get_transaction(self, wallet_id: str, transaction_id: bytes32) -> Tran
     async def get_transactions(
         self,
         wallet_id: str,
+        all: Optional[bool] = None,
```

Comment:
I think it would be reasonable to require all these parameters to be named, by inserting a `,*` here.
Especially since there are 3 plain `ints` in a row. If a caller is allowed to specify those as positional arguments, you can't tell which one is which

Reply:
Agreed. and changed